fastlane/testflight.md

```diff
@@ -2,21 +2,26 @@
 
 These instructions allow you to build Trio without having access to a Mac.
 
-* You can install Trio on phones using TestFlight that are not connected to your computer
-* You can send builds and updates to those you care for
-* You can install Trio on your phone using only the TestFlight app if a phone was lost or the app is accidentally deleted
-* You do not need to worry about specific Xcode/Mac versions for a given iOS
+- You can install Trio on phones using TestFlight that are not connected to your computer
+- You can send builds and updates to those you care for
+- You can install Trio on your phone using only the TestFlight app if a phone was lost or the app is accidentally deleted
+- You do not need to worry about specific Xcode/Mac versions for a given iOS
 
 ## **Automatic Builds**
-> 
+
 > The browser build defaults to automatically updating and building a new version of Trio according to this schedule:
+>
+> <<<<<<< HEAD
```

Comment:
Looks like some kind of merge conflict crept in here.
When I tested a couple of weeks ago, I was able to do a `git apply` by removing the `testflight.md` portion from the patch and then forgot I had done that.

Reply:
Fixed 👍